lib/reporter/results.js

```diff
@@ -21,6 +23,11 @@ module.exports = class Results {
       assertions: [],
       tests: 0
     };
+    // Adding sessionInfo to reporter
+    this.driverCapabilities = {};
```

Comment:
what is this used for? also, it's only the session that has capabilities, the driver is just a connector.

Reply:
Test Observability require capabilities to be displayed on dashboard. We would also need it for html reporter v2 to display environment specific details.